modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -3551,7 +3557,7 @@
                         ],
                         "federalActivation": {
                           "anticipatedSeparationDate": "10-03-2025",
-                          "activationDate": "02-11-2018"
+                          "activationDate": "02-11-2024"
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/14093/files#r1355237794 same as above.. just confirming.